optaweb-employee-rostering-frontend/src/ui/components/ServerSideExceptionDialog.tsx

```diff
@@ -48,10 +48,21 @@ export const ServerSideExceptionDialog: React.FC<React.PropsWithChildren<ServerS
   const [isOpen, setIsOpen] = React.useState(false);
   const dialogBody = createStackTrace(props);
   const { t } = useTranslation('ServerSideException');
+  let messageParameters: any;
```

Comment:
Can be `const` if you use ternary operator on the right hand side of the assignment. Also would be nice to avoid `any`. If you do that, you'll see there is a type error in the `t()` call on line 65. It might be worth investigating.

Reply:
After investigation, apparently formatting options go into the same object as interpolaration parameters. the second parameter is for the default value if a translation is not found.